.gitignore

```diff
@@ -0,0 +1,2 @@
+#dependencies
+/venv
```

Comment:
`.gitignore` files are commonly used to avoid committing and pushing files or directories to GitHub that are either sensitive or unnecessary, such as dependencies